Daramola98/WEconnectApp
# #158723547 Implement feature user should be able to rate a business

#### What does this PR do?
This PR implement feature user should be able to rate a business

#### Description of Task to be completed?
- Add rating to the review model
- Add rating to controllers
- Integrate react stars for rating businesses
- Integrate rating to UI

#### How should this be manually tested?
- Setup project following instructions in readme then run the command npm run start:dev
- Open postman and hit the route to review a business make sure you are authenticated, you would be able to add a rating for a business

#### What are the relevant pivotal tracker stories?
#158723547
